vector/src/main/AndroidManifest.xml

```diff
@@ -242,7 +243,7 @@
                 <data
                     android:host="room"
                     android:scheme="element" />
-
+                <data android:scheme="schildichat" />
```

Comment:
Why not restrict to `room` and `user` like upstream does? Furthermore, this likely conflicts with the one above since you haven't restricted the SSO one either?

Reply:
All of the <data> tags are merged together, so the android:host attributes from above apply also to schildichat.  I can add the same ```android:host="room"``` tags as above if you want but they would have no effect and I want to keep fewer lines changed here.

see:
https://developer.android.com/training/app-links/deep-linking#adding-filters

Reply:
Ah, good to know! Then that's fine